### PR TITLE
fix: [select-dialog] not move to center

### PR DESCRIPTION
**Pull request by GongHeng2017**
1. add move to center logic code

Log: fix issue
Bug: https://pms.uniontech.com/bug-view-202177.html

### CODE REVIEW

File: src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp [hunk 1]
```diff
@@ -1040,11 +1038,6 @@ void FileDialog::initializeUi()
 #endif
     statusBar()->lineEdit()->lineEdit()->setMaxLength(NAME_MAX);
 
-    // 修复bug-45176
-    // 如果是wanyland平台，将弹出的文件框居中
-    if (WindowUtils::isWayLand())
```
Comment (max-lvs) [inline src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp:1045]: 为什么直接删了呢？

**Review by max-lvs — APPROVED** [review]

**Review by Lighto-Ku — APPROVED** [review]

File: src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp (hunk 1 above)
Comment (GongHeng2017) [reply src/plugins/filedialog/filedialogplugin-core/views/filedialog.cpp:1045]: 重新实现了居中效果，

d->dialog->moveCenter(WindowUtils::cursorScreen()->availableGeometry().center());

**Pull request merged**